src/pseudopeople/entity_types.py

```diff
@@ -50,6 +52,10 @@ class ColumnNoiseType:
 
     name: str
     noise_function: Callable[[pd.Series, ConfigTree, RandomnessStream, Any], pd.Series]
+    row_noise_level: float = 0.01
+    token_noise_level: float = 0.1
+    additional_parameters: Dict = None
```

Comment:
Nit: `Dict[str, Any]`? Or `Dict[str, Union[float, Dict]]`?

Reply:
Thoughts? Just making sure you saw this.

Reply:
Oops, I did miss this